docs/mods/fast_guage.md

```diff
@@ -0,0 +1,18 @@
+# Fast Guage
+
+This is a quick tutorial on how to modify the guage of exp and health bars of Plat so that the bar visually updates in subpixels per second instead of points per second.
+
+This mod is a backport of the healthbar fix made by the contributors of [pret/pokeheartgold](https://github.com/pret/pokeheartgold) for the heartgold/soulsilver decomp.
```

Comment:
**style**: Provide a perma-link to the original fix in `pokeheartgold`.

Reply:
I fixed the typos and provided a direct link to the line that's changed in pret/pokeheartgold.